Formula/box2d.rb

```diff
@@ -0,0 +1,30 @@
+class Box2d < Formula
+  desc "2D physics engine for games"
+  homepage "https://box2d.org"
+  url "https://github.com/erincatto/box2d/archive/v2.4.0.tar.gz"
+  sha256 "38bce8217afb974e5be070da0b6768bb4fc0e246fa1cee42fecf71c4c6e9ad19"
+  license "MIT"
+  head "https://github.com/erincatto/Box2D.git"
+
+  depends_on "cmake" => :build
+
+  def install
+    args = std_cmake_args + %w[
+      -DBOX2D_BUILD_UNIT_TESTS=OFF
+      -DBOX2D_BUILD_TESTBED=OFF
+      -DBOX2D_BUILD_EXAMPLES=OFF
+    ]
+
+    system "cmake", ".", *args
+    system "cmake", "--build", "."
+    include.install Dir["include/*"]
+    lib.install "src/libbox2d.a"
```

Comment:
Is there no `make install` target to handle this? Manual installation will break if upstream adds or removes things in the future.

Reply:
Nope, there is not.

Reply:
Please open an issue upstream asking for it, and include the link with a comment in the formula.

Reply:
There was a `cmake --install` recently added to that lib. I've opened an issue asking for a new release (https://github.com/erincatto/box2d/issues/649), hope it can be released soon, if not, I'll update the formula with your suggested changes and also will link this issue here so this can be merged.